class.jetpack.php

```diff
@@ -4886,6 +4892,18 @@ public static function clean_nonces( $all = false ) {
 		}
 	}
 
+	public static function clean_exports() {
+		$upload_dir = wp_upload_dir();
+		$export_glob = $upload_dir['path'] . '/export_*xml';
+		$ten_minutes_ago = time() - 600;
+		foreach ( glob( $export_glob ) as $export_filename ) {
```

Comment:
As a compat measure for servers that don't have `glob` available (at our scale, we hit some strange configurations) we prefer to use [`Jetpack::glob_php()`](https://github.com/Automattic/jetpack/blob/1698ea54bef74cec145ba3c10415a87dee5ee268/class.jetpack.php#L1857-L1892) for PHP files -- now as you're dealing with XML files here and not PHP that won't work directly, but it may be better to follow the conventions there to provide a fallback.